docs/examples/zoom.ipynb

```diff
@@ -0,0 +1,285 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "ba4d302c",
+   "metadata": {},
+   "source": [
+    "# Zoom data reduction\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "This notebook is an example of how ESSsans can be used to reduce data from [Zoom at ISIS](https://www.isis.stfc.ac.uk/Pages/Zoom.aspx).\n",
+    "The following description is kept relatively brief, for more context see the rest of the documentation.\n",
+    "In particular the [Sans2d](./sans2d.ipynb) notebook may be useful."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "53e2278d",
+   "metadata": {},
+   "source": [
+    "There are a few things that are not yet handled:\n",
+    "\n",
+    "- TOF or wavelength masks\n",
+    "- Position corrections from user file"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09234b87",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "### Imports and configuration"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "319162e9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "import sciline\n",
+    "import scippneutron as scn\n",
+    "import plopp\n",
+    "import esssans as sans\n",
+    "import esssans.isis\n",
+    "from esssans.types import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "64783c51",
+   "metadata": {},
+   "source": [
+    "### Setup input files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9e7cfc82",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "params = {\n",
+    "    sans.types.DirectBeamFilename: 'Direct_Zoom_4m_8mm_100522.txt',\n",
+    "    sans.isis.CalibrationFilename: '192tubeCalibration_11-02-2019_r5_10lines.nxs',\n",
+    "    sans.isis.Filename[sans.types.SampleRun]: 'ZOOM00034786.nxs',\n",
+    "    sans.isis.Filename[sans.types.EmptyBeamRun]: 'ZOOM00034787.nxs',\n",
+    "}\n",
+    "masks = [\n",
+    "    'andru_test.xml',\n",
+    "    'left_beg_18_2.xml',\n",
+    "    'right_beg_18_2.xml',\n",
+    "    'small_bs_232.xml',\n",
+    "    'small_BS_31032023.xml',\n",
+    "    'tube_1120_bottom.xml',\n",
+    "    'tubes_beg_18_2.xml',\n",
+    "]\n",
+    "masks = sciline.ParamTable(str, {sans.isis.PixelMaskFilename: masks}, index=masks)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a8a4e01",
+   "metadata": {},
+   "source": [
+    "### Setup reduction parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "50f11eb2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "params[NeXusMonitorName[Incident]] = 'monitor3'\n",
+    "params[NeXusMonitorName[Transmission]] = 'monitor5'\n",
+    "\n",
+    "band = sc.linspace('wavelength', 1.75, 16.5, num=2, unit='angstrom')\n",
+    "params[WavelengthBands] = band\n",
```

Comment:
Would be good if we could leave the `WavelengthBands` and only have to specify the `WavelengthBins`?

Reply:
I copied this from existing notebooks, not sure what you are suggesting.

Reply:
Just that seeing it again here makes me think we could improve the usability a little more. I think it's a bug, and it's unrelated to this PR, so let's not do anything about it here.

Reply:
See #51